pubspec.yaml

```diff
@@ -3,14 +3,19 @@ description: A thin wrapper around GrowthBook's Flutter SDK to support automated
 version: 1.0.0
 homepage: https://github.com/uptech/uptech-growthbook-sdk-flutter
 
+publish_to: none
+
 environment:
   sdk: '>=2.18.1 <3.0.0'
   flutter: '>=1.17.0'
 
 dependencies:
   flutter:
     sdk: flutter
-  growthbook_sdk_flutter: ^2.0.0
+  growthbook_sdk_flutter:
+   git:
+      url: https://github.com/uptech/GrowthBook-SDK-Flutter.git
+      ref: master 
```

Comment:
We'll see if we need to go this direction based on the responsiveness of the maintainers of the GrowthBook-SDK-Flutter.